columnflow/production/cms/btag.py

```diff
@@ -147,14 +192,18 @@ def add_weight(syst_name, syst_direction, column_name):
             # apply to all but c flavor
             flavor_mask = flavor != 4
 
+        # prepare arguments
+        args = {
```

Comment:
this dict is currently named `variable_map` in most of our correctionlib-based Producers. Would be nice to stay consistent here, but not necessary

Reply:
True. I renamed it to be consistent.